fabric-networking-v0/src/main/java/net/fabricmc/fabric/impl/network/FabricNetworkInitializer.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.network;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import net.fabricmc.api.ModInitializer;
+import net.fabricmc.fabric.api.util.NbtType;
+import net.fabricmc.fabric.impl.network.login.S2CLoginHandshakeCallback;
+import net.minecraft.client.network.packet.LoginDisconnectS2CPacket;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.text.LiteralText;
+import net.minecraft.text.Text;
+
+public class FabricNetworkInitializer implements ModInitializer {
+	protected static final Logger LOGGER = LogManager.getLogger();
+    
+    public static final LiteralText MISMATCH_VERSION_TEXT = new LiteralText("Mismatched hello packet version, expected version: " + FabricHelloPacketBuilder.VERSION_MAJOR + "." + FabricHelloPacketBuilder.VERSION_MINOR);
```

Comment:
TranslateableText plz

Reply:
For that case I could add a lang file for the networking extension and be fine.

But further down is a vanilla message that has to be literal because the vanilla client can receive new lang entrys via packets.